WordPress/Classes/ViewRelated/Blog/DiscussionSettingsViewController.swift

```diff
@@ -66,6 +65,18 @@ public class DiscussionSettingsViewController : UITableViewController
     
 
     // MARK: - Persistance!
+    private func refreshSettings() {
+        let service = BlogService(managedObjectContext: settings.managedObjectContext)
+        service.syncSettingsForBlog(settings.blog,
```

Comment:
Nice catch! This fellow was initially only storing the `settings` instance. Down the line that was changed, and this is proof of it.

Thank you!!!